pkg/services/rpcsrv/server.go

```diff
@@ -2449,21 +2446,35 @@ func getRelayResult(err error, hash util.Uint256) (any, *neorpc.Error) {
 		return result.RelayResult{
 			Hash: hash,
 		}, nil
-	case errors.Is(err, core.ErrAlreadyExists):
+	case errors.Is(err, core.ErrTxExpired):
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrExpiredTransaction, err.Error())
+	case errors.Is(err, core.ErrAlreadyExists) || errors.Is(err, core.ErrHasConflicts) || errors.Is(err, core.ErrInvalidBlockIndex):
 		return nil, neorpc.WrapErrorWithData(neorpc.ErrAlreadyExists, err.Error())
+	case errors.Is(err, core.ErrAlreadyExistsMemPool):
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrTxAlreadyInPool, err.Error())
 	case errors.Is(err, core.ErrOOM):
-		return nil, neorpc.WrapErrorWithData(neorpc.ErrOutOfMemory, err.Error())
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrMempoolCapReached, err.Error())
```

Comment:
To get `ErrAlreadyExistsMemPool` you have to submit the same transaction twise. The firs time it should successfully pass all checks and be added to the mempool, the second time you'll get `ErrAlreadyInPool`.

Reply:
To get `ErrOOM` you have to configure a small mempool in our RPC server (use the `MemPoolSize` protocol configuration setting), submit a set of transactions so that the mempool reaches its capacity and send one more transaction that is less prioritized than others. You'll get `ErrOOM` afterwards.